app/controllers/books_controller.rb

```diff
@@ -5,5 +5,11 @@ def index
 
   def show
     @book = Book.find(params[:id])
+    @reviews = Review.where(book_id: @book.id).order("created_at DESC")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.